pipeline_plugins/resource/hook/celaction.py

```diff
@@ -9,8 +9,7 @@
 import _winreg
 
 if __name__ == "__main__":
```

Comment:
We don't actually need this when run through ftrack-connect.
Are you using this?